### ISSUE TITLE
Not-an-issue-but-a-question: Is TabDAE an unsupervised autoencoder? 

### ISSUE THREAD

**Issue by LuisLauM** [Ransaka/Tab-DAE#1]
Hi. Thank you very much for sharing this tool. Reviewing the Jupyter notebook code and README, I notice that TabDAE does not explicitly use `train_labels` (nor `test_labels`) for training. So, I would like to know if this is because it is a non-supervised autoencoder or maybe it does use them but it is some internal stage. Thank you.

**Comment by Ransaka** [comment]
Hi, Since this is an autoencoder, it tries to reconstruct inputs with minimum loss. Therefore, you can see in the training loop that i have used `input` to generate predictions. As the model tries to reconstruct the `input`, the model `output` should be similar to the `input` (when the model finishes training). In the backward pass, inside the training loop, it compares the original `input` and its reconstructed form (model output) and produces a single number as a measure of the model's performance, called the loss.

**Comment by LuisLauM** [comment]
Thank you. So, the Tab_DAE itself and its optimization process do not need (nor use) the output labels (0-9) but only the input and output flatened matrices (i.e. vectors), right?

**Comment by Ransaka** [comment]
yes, that's correct.

**Comment by LuisLauM** [comment]
Thank you so much!
